contentcuration/contentcuration/frontend/channelEdit/components/PublishModal/ChannelMetadata.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+
+  <div>
+    <VProgressCircular v-if="loading" indeterminate size="16" color="grey lighten-1" />
```

Comment:
We have a `<KCircularLoader>` component that might be good to make use of here while we're at the business of introducing KDS. The API is pretty simple anyway.